index.html

```diff
@@ -2073,10 +2090,10 @@ <h2>All your favourite systems</h2>
       <div class="content">
         <div class="gridrow">
           <div class="col_6">
-            <h2>Standard Hotkeys</h2>
+            <h2>Hotkeys Galore</h2>
             <p class="large">
-              We follow the AmberElec convention so you can jump right in and know how to save games. And in some
-              emulators we unleash all the Steam Deck's extra controls using custom Steam Input Profiles.
+              EmuDeck follows the AmberElec convention so you can jump right in and button combo like an expert. In some
+              emulators, we unleash all the Steam Deck's extra controls using custom Steam Input Profiles. For a full list of the hotkeys EmuDeck utilizes, visit the <a target="_blank" href="https://github.com/dragoonDorise/EmuDeck/wiki/Hotkeys">hotkey page</a> on the EmuDeck Wiki.
```

Comment:
@rawdatafeel please remove all links in the descriptions, I don't want people to get too much information and get lost in the process